addons/website_slides/views/website_slides_templates_profile.xml

```diff
@@ -17,14 +17,17 @@
                     <t t-if="courses_completed" t-call="website_slides.display_course">
                         <t t-set="courses" t-value="courses_completed"></t>
                     </t>
-                    <p t-else="" class="text-muted">No completed course yet</p>
+                    <p t-else="" class="text-muted">No completed courses yet!</p>
+                    <div class="text-right">
+                        <a href="/slides/all" class="btn btn-link btn-sm"><i class="fa fa-arrow-right mr-1"/>Check all available courses</a>
+                    </div>
```

Comment:
If there are no completed courses yet (or badges, or certifications), it takes a lot of page space and feels out of balance (IMO).

I would go for:

```
<div t-else="" class="text-muted d-inline-block">No completed courses yet!</p>
<div t-attf-class="text-right #{'d-inline-block pull-right' if not courses_completed else ''}">
    <a href="/slides/all" class="btn btn-link btn-sm"><i class="fa fa-arrow-right mr-1"/>Check all available courses</a>
</div>
```

Same for others.

Feel free to test it and tell me if it does not look good to you.